source/Fleet.h

```diff
@@ -98,7 +98,7 @@ class Fleet {
 	// The number of different items the ships in this fleet will carry in cargo.
 	int cargo = 3;
 	std::vector<std::string> commodities;
-	std::set<const Sale<Outfit> *> outfitters;
+	std::set<const OutfitSale *> outfitters;
```

Comment:
we could want merchants for example to not use hidden items, and also because OutfitSale is contained within GameData we would have to modify it, converting OutfitSale to Sale
that's what I was doing originally but quyyk advised against it, to have all classes use OutfitSale when it is about outfit